packages/docs/docs/features/device-settings.mdx

```diff
@@ -22,15 +22,26 @@ The opened device settings dropdown looks as follows:
 2. **Text Size** - a slider that allows to change the font size on the device.
 3. **Press Home Button** - simulates a press of the home button.
 4. **Open App Switcher** - allows to quickly switch between opened apps.
-5. **Biometrics** - options for simulating biometric authorization on iOS. This option isn't available on Android. The following options are available:
+5. **Rotate Device** - allows to rotate the device or set its orientation. 
+
+<video autoPlay loop width="700" controls className="shadow-image">
+  <source src="/video/ide_rotation.mp4" type="video/mp4"/>
+</video>
+
+Available orientations are as follows:
```

Comment:
please also describe them in text as it's impossible to search with cmd+f, algolia or even google when it's only available as screenshot